src/Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM python:3.7.0
+FROM python:3.11.3
 
 WORKDIR /usr/src/
 
-RUN apt-get update && apt-get -y install jq
-
```

Comment:
I still think you want to keep this.  It might be up to date today, but it may not be tomorrow.

Reply:
This image is just for running the tests right? If so I don't believe jq is used at all